examples/satellite-azure/inputs.tfvars.template

```diff
@@ -0,0 +1,32 @@
+##############################################################################
+## IBM Cloud Variables
+##############################################################################
+# Set the variable export TF_VAR_ibmcloud_api_key=
+#ibmcloud_api_key = ""
+
+
+##############################################################################
+## Azure Variables
+##############################################################################
+client_id       = ""
+tenant_id       = ""
+client_secret   = ""
+subscription_id = ""
+
+# Standard_D16ds_v4: 16 vCPU - 64 GB AM
+instance_type = "Standard_D16ds_v4"
+
+
+##############################################################################
+## Satellite Variables
+##############################################################################
+managed_from   = "wdc"
+location_zones = ["eastus-1", "eastus-2", "eastus-3"]
+
+
+##############################################################################
+## ROKS Cluster
+##############################################################################
+
+# Total number of worker nodes for cluster
+addl_host_count = 3
```

Comment:
this will still work for now, but i'd recommend using the new `addl_hosts` json style w/ the flavor in there.

Reply:
Done!